R/lifecycle-package.R

```diff
@@ -7,3 +7,7 @@
 ## usethis namespace: start
 ## usethis namespace: end
 NULL
+
+.onLoad <- function(libname, pkgname) {
+  backports::import(pkgname, "strrep")
```

Comment:
Might be worth just implementing in `utils.R` because it'll avoid a dependency and it's pretty simple anyway.